apps/src/templates/teacherDashboard/EditSectionForm.story.jsx

```diff
@@ -1,74 +1,85 @@
 import React from 'react';
 import {UnconnectedEditSectionForm as EditSectionForm} from './EditSectionForm';
 import {action} from '@storybook/addon-actions';
-import {SectionLoginType} from '@cdo/apps/util/sharedConstants';
 import {testSection, courseOfferings} from './teacherDashboardTestHelpers';
 
-export default storybook => {
-  storybook = storybook.storiesOf('EditSectionForm', module);
+export default {
+  title: 'EditSectionForm',
+  component: EditSectionForm
+};
+
+const noStudentsSection = {
+  ...testSection,
+  studentCount: 0,
+  participantType: 'student'
+};
+
+const studentSection = {
+  ...testSection,
+  participantType: 'student'
+};
+
+const emailLogin = {
+  ...testSection,
+  loginType: 'email',
+  participantType: 'student'
+};
+
+const wordLogin = {
+  ...testSection,
+  loginType: 'word',
+  participantType: 'student'
+};
+
+const pictureLogin = {
+  ...testSection,
+  loginType: 'picture',
+  participantType: 'student'
+};
+
+const Template = args => (
+  <EditSectionForm
+    title="Edit section details"
+    handleSave={action('handleSave')}
+    handleClose={action('handleClose')}
+    editSectionProperties={action('editSectionProperties')}
+    courseOfferings={courseOfferings}
+    sections={{}}
+    hiddenLessonState={{}}
+    updateHiddenScript={() => {}}
+    assignedUnitName="script name"
+    assignedUnitLessonExtrasAvailable={false}
+    assignedUnitTextToSpeechEnabled={false}
+    {...args}
+  />
+);
+
+export const GenericEmail = Template.bind({});
+GenericEmail.args = {
+  section: emailLogin,
+  isSaveInProgress: false
+};
+
+export const GenericWord = Template.bind({});
+GenericWord.args = {
+  section: wordLogin,
+  isSaveInProgress: false
+};
+
+export const GenericPicture = Template.bind({});
+GenericPicture.args = {
+  section: pictureLogin,
+  isSaveInProgress: false
+};
+
+export const NoStudents = Template.bind({});
+NoStudents.args = {
+  section: noStudentsSection,
+  isSaveInProgress: false
+};
 
-  Object.values(SectionLoginType).forEach(loginType => {
```

Comment:
It looks like [SectionLoginType](https://github.com/code-dot-org/code-dot-org/blob/570c266367463e0e0d2c9a8717d1289648257eae/lib/cdo/shared_constants.rb#L43) also includes 'clever' and 'google_classroom' that are missing from the translation. Intentional?

Reply:
Thanks for this- There was a loop before that was a little less clear how to replicate with the new format so I missed these. Updated!